openmmtools/alchemy.py

```diff
@@ -2038,6 +2060,112 @@ def _alchemically_modify_AmoebaMultipoleForce(self, reference_force, alchemical_
             alchemical_region = alchemical_region[0]
         raise Exception("Not implemented; needs CustomMultipleForce")
 
+    def _alchemically_modify_CustomBondForce(self, reference_force, alchemical_regions, _):
+        energy_function = reference_force.getEnergyFunction()
+
+        # this is a quick fix.
+        # TODO: extend to multiple alchemical regions
+        assert len(alchemical_regions) == 1
```

Comment:
Could you raise a `NotImplementedError` here rather than an assert explaining this is not currently supported?